python/pyarrow/tests/test_parquet.py

```diff
@@ -89,22 +83,21 @@ def _roundtrip_pandas_dataframe(df, write_kwargs):
     return table1.to_pandas()
 
 
-@parquet
```

Comment:
Why did you remove the decorators instead of switching to the new name?

Reply:
Because I've switched to a module level marker instead